src/util/shared-util.ts

```diff
@@ -63,3 +65,23 @@ export const escapeDiscord = function (message: string): string {
 
   return message
 }
+
+export function filterProfanity(
+  playerMessage: string,
+  app: Application
```

Comment:
limit the scope of the argument to only requiring `profanityFilter` instead of the entire application.